code/Compass.php

```diff
@@ -310,9 +321,20 @@ public function rebuild($verbose = false)
                     continue;
                 }
                 
+                // Skip other modules
+                if ($name == 'framework') {
```

Comment:
I don't know why, but framework's Sass threw some compiling errors, so I thought, you need some more Compass plugins or external mixins to compile. However, I've tested this now without skipping core modules – no compile errors :)